src/client/common/components/base-network-view/base-network-view.js

```diff
@@ -70,10 +72,10 @@ class BaseNetworkView extends React.Component {
 
       const layout = cy.layout(initialLayoutOpts);
 
-      //set default view for filtering nodes by p_value when new network is rendered
       this.state.cySrv.loadPromise().then(() => {
-        //by default, only display nodes with p_value <= 0.025
-        this.displayDefaultNodes(0.025);
+      //by default, display all nodes (p-value <= .05)
+      //match position on slider to that of default value
+      document.getElementById('enrichment-selection-slider').value = 0.05;
```

Comment:
It would be preferable to set the value as an attribute in render().  React allows a defaultValue, if you only need it set initially.